app/uk/gov/hmrc/apidocumentation/views/openapispec/ParentPageOuter.scala.html

```diff
@@ -0,0 +1,68 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.mvc.Request
+@import uk.gov.hmrc.apidocumentation.config._
+@import uk.gov.hmrc.apidocumentation.controllers
+@import uk.gov.hmrc.apidocumentation.models._
+@import uk.gov.hmrc.apidocumentation.views.apispecification.helpers._
+@import uk.gov.hmrc.apidocumentation.views.helpers.VersionDocsVisible
+@import uk.gov.hmrc.apidocumentation.views.html.include._
+@import uk.gov.hmrc.apidocumentation.models.apispecification.DocumentationItem
+
+@this(apiMain: apiMain, parentPageView: ParentPageView)
+
+@(pageAttributes: PageAttributes, markdownBlocks: List[DocumentationItem], title: String, api: ExtendedAPIDefinition, currentVersion: ExtendedAPIVersion, loggedIn: Boolean = false, deprecationMessage: Option[String] = None)(implicit applicationConfig: ApplicationConfig, request: Request[AnyContent], messagesProvider: MessagesProvider)
+
+@topContent = {
+    <div class="form-group">
+        <select class="form-control form-control-version"
+                id="version"
+                name="version"
+                value="@controllers.routes.ApiDocumentationController.renderApiDocumentation(api.serviceName, currentVersion.version, None).url"
+                onchange="this.options[this.selectedIndex].value && (window.location = this.options[this.selectedIndex].value);">
+            @for(v <- api.sortedActiveVersions;
+                if VersionDocsVisible(v.visibility) != DocsVisibility.NOT_VISIBLE) {
+                <option @if(v == currentVersion) {selected} value="@controllers.routes.ApiDocumentationController.renderApiDocumentation(api.serviceName, v.version, None).url">v@v.version (@v.displayedStatus)</option>
```

Comment:
missing 'aria-label' attribute? (there is one on the same elements on the page below)